# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/hipcc/hotfix-lib-llvm.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- hipcc/src/hipBin_amd.h.vanilla	2025-04-01 20:10:36.937809670 +0200
+++ hipcc/src/hipBin_amd.h	2025-04-01 20:11:13.898331954 +0200
@@ -209,7 +209,7 @@
     } else {
       compilerPath = getRoccmPath();
       hipClangPath = compilerPath;
-      hipClangPath /= "lib/llvm/bin";
+      hipClangPath /= "bin";
     }
 
     compilerPath = hipClangPath.string();
